person.rb

```diff
@@ -0,0 +1,14 @@
+class Person
+  attr_reader :age, :phase
+  attr_accessor :name
+
+  def initialize(options = {})
+   @name = options.fetch(:name, "")
+   @age = options.fetch(:age, 0) 
+  end
+
+  def offer_high_five
+    "High five!"
+  end
+
```

Comment:
This method can go into a separate module called "HighFiveable".

Usually for modules that are going to be included in classes, we use "-able" names, so here it'd be HighFiveable.